WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -560,6 +562,19 @@ private extension CardPresentPaymentsOnboardingUseCase {
                                                                                 error: error,
                                                                                 gatewayID: preferredPluginLocal?.gatewayID))
     }
+
+    func logMissingRequirements(for account: PaymentGatewayAccount) {
+        let log = """
+            ❌ Account has missing requirements:
+            Gateway ID: \(account.gatewayID)
+            Account status: \(account.status)
+            WCPay status: \(account.wcpayStatus)
```

Comment:
While these two should be the same, I though was worth to log them separately in case that there's any conflict 🤔